R/XcmsExperiment.R

```diff
@@ -1,4 +1,5 @@
-#'@title Methods to save and load contents of a XcmsExperiment object
+#' @include PlainTextParam.R
```

Comment:
Yes, if not specified/modified with `@include` source files are ordered alphabetically. So, it would be OK for this to not include the `@include` - but I wanted to be explicit.